src/components/CompatList.astro

```diff
@@ -0,0 +1,39 @@
+---
+import compatTable from '../../pluginsCompatTable.json';
+
+type PluginTable = {
+	name: string;
+	platforms: string[];
+};
+
+interface Props {
+	plugin: string;
+}
+// TODO: Find proper yellow color scheme with text, preferably with starlight variables
+
+let { plugin } = Astro.props;
+const platformList: PluginTable | undefined = compatTable.find((p) => p.name === plugin);
+const platforms = platformList ? platformList.platforms : [];
+---
+
+<div class="flex row not-content">
+	{platforms.map((platform) => <p>{platform}</p>)}
```

Comment:
I think that adding the icons next to the name/platform would be even better, what you guys think? @tauri-apps/wg-doc